__tests__/artist-create.test.js

```diff
@@ -0,0 +1,35 @@
+const { expect } = require('chai');
+const request = require('supertest');
+
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create artist', () => {
+  let db;
+  beforeEach(async () => (db = await getDb()));
+
+  afterEach(async () => {
+    await db.query('DELETE FROM Artist');
+    await db.close();
+  });
+
+  describe('/artist', () => {
+    describe('POST', () => {
+      it('creates a new artist in the database', async () => {
+        const res = await request(app).post('/artist').send({
+          name: 'Tame Impala',
+          genre: 'rock'
+        });
+
+        expect(res.status).to.equal(201);
```

Comment:
This assertion could be moved to be with the others at the end of the test for uniformity. I mentioned above that the detail of the assertion wouldnt be included in the output and it doesnt necessarily need to be here, though I see the reasoning for it being here as it could still be useful for sanity checking the result. Just something to consider, as if the query did not work then the assertions at the end of the test would fail